homeassistant/components/aladdin_connect/sensor.py

```diff
@@ -89,14 +98,17 @@ def __init__(
         device: DoorDevice,
         description: AccSensorEntityDescription,
     ) -> None:
-        """Initialize a sensor for an Abode device."""
+        """Initialize a sensor for an Aladdin Connect device."""
         self._device_id = device["device_id"]
         self._number = device["door_number"]
         self._name = device["name"]
+        self._model = device["model"]
         self._acc = acc
         self.entity_description = description
         self._attr_unique_id = f"{self._device_id}-{self._number}-{description.key}"
         self._attr_has_entity_name = True
+        if self._model == "01" and description.key != "rssi":
+            description.entity_registry_enabled_default = True
```

Comment:
This should be part of the entity description itself?

Reply:
For 99% of the models, these are disabled by default.  For this ONE model, two of the three should be enabled when integration is added. 
Is there another way to do that?